cms/templates/howitworks.html

```diff
@@ -14,8 +16,8 @@
 <div class="wrapper-content-header wrapper">
   <section class="content content-header">
     <header>
-      <h1><span class="wrapper-text-welcome">${_("Welcome to {studio_name}").format(
-        studio_name=u'</span><span class="logo">{studio_name}</span>'.format(
+      <h1><span class="wrapper-text-welcome">${Text(_("Welcome to {studio_name}")).format(
+        studio_name=HTML(u'</span><span class="logo">{studio_name}</span>').format(
```

Comment:
Having `studio_name` start with `</span>` doesn't make sense and could break i18n.  The cleanest solution would be if `<span class="wrapper-text-welcome">` could wrap the entire message.  This may require tuning of the `logo` class or an additional class to override `wrapper-text-welcome` if needed, but it would make the translation much more flexible.